Sources/ApolloCodegenLib/Templates/SelectionSetTemplate.swift

```diff
@@ -748,15 +748,10 @@ fileprivate struct SelectionSetNameGenerator {
 fileprivate extension IR.ScopeCondition {
 
   var selectionSetNameComponent: String {
-    if let type = type {
-      return "As\(type.name.firstUppercased)"
-    }
-
-    if let conditions = conditions {
-      return "If\(conditions.typeNameComponents)"
-    }
-
-    fatalError("ScopeCondition is empty!")
```

Comment:
Would the missing scope condition have been an internal error? If this is encountered again the generated code probably wouldn't build but how would we be able to narrow down on this being the source of that?